doc/CodeDocumentation.conf.in

```diff
@@ -1799,7 +1805,7 @@ MATHJAX_VERSION        = MathJax_2
 # The default value is: HTML-CSS.
 # This tag requires that the tag USE_MATHJAX is set to YES.
 
-MATHJAX_FORMAT         = HTML-CSS
+MATHJAX_FORMAT         = SVG
```

Comment:
Is SVG better?

Reply:
For me, the rendering with `HTML-CSS` did not look good (on Mac/Chrome). Will be good to get data from other folks.

Reply:
Does this mean SVGs are generated by `doxygen` for every formula, or that the SVGs are generated only in-browser by mathjax to render the formulas? If the former is true, then I think we should revert this -- having a ton of `.svg` files for every formula seems quite excessive.

It looks like this may control the "MathJax output format": https://docs.mathjax.org/en/latest/output/index.html, so SVGs are probably generated on-the-fly in the browser.

Maybe we can tweak some other MathJax option (e.g. the math font, if that's part of the doxygen-generated html) to get nice looking "HTML-CSS". Or, maybe we can try MathJax v3 instead of v2, see https://www.doxygen.nl/manual/config.html#cfg_use_mathjax. According to these docs, "HTML-CSS (which is slower, but has the best compatibility. ...)" so I guess that's why it's the default.

Reply:
This is something I will appreciate some help with (to make sure it is not due to my setup/browser/OS).

If you have time, can you try the different MathJax output formats and see if you can use `MathJax_3`?

For me `HTML-CSS` did not look good and `MathJax_3` didn't work on a first try.

Reply:
Here's what `HTML-CSS` looks like for me (compare with the image in the PR description)

<img width="837" alt="Screenshot 2024-02-01 at 8 33 56 AM" src="https://github.com/mfem/mfem/assets/1278247/01b5e203-7836-4cb1-a603-044c06eca488">

Reply:
Here is what `MATHJAX_VERSION = MathJax_3` plus `MATHJAX_FORMAT = chtml` looks like:
![Uploading Screenshot 2024-02-01 at 8.39.06 AM.png…]()

Reply:
Switched to MathJax3 in 44d33d17f32377ab9fbea204b57e5aa16d0a40d5 -- thanks @najlkin !